.eslintrc.yml

```diff
@@ -0,0 +1,105 @@
+env:
+  es6: true
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+plugins:
+  - react
```

Comment:
Does Madeline have react? or is this a leftover from NEMO?

Reply:
some leftovers, yeah, thanks